### DISCUSSION TITLE
Is there a way to know that user was created through supabase Auth web-ui?

### DISCUSSION THREAD

**Discussion by Bogdan1001** [supabase/supabase#17509] [Questions]
We're building a role-based system.

Is there a way to know that the user was created through supabase Auth web-ui? and not from any other place.

And additionally in which moment "confirmed_at" timestamp inserted in auth.users table when we select authoconfirm in supabase Auth web-ui?

**Comment by GaryAustin1** [comment]
What to you mean by Auth web-ui?   The dashboard auth section?

If so, you might be able to look at the auth.users table and look at difference between created_at and email_confirmed_at columns IF you require confirmation for email normally.   In the non dashboard autoconfirm case it would have a difference of more than a second.

**Comment by Bogdan1001** [comment]
Yes, the dashboard auths section just when we apply the trigger:

` after insert on auth.users
`

both confirmation dates are null in NEW.  So we've created an additional trigger to solve this.

` AFTER UPDATE OF confirmed_at ON auth.users`

Can I also somehow check on the user creation that a user was created through the dashboard auths section and not any other way?
